seed/static/seed/js/controllers/data_upload_modal_controller.js

```diff
@@ -64,7 +64,7 @@ angular.module('BE.seed.controller.data_upload_modal', [])
       organization
     ) {
       $scope.cycles = cycles.cycles;
-      if ($scope.cycles.length) $scope.selectedCycle = $scope.cycles[0];
+      if ($scope.cycles.length) $scope.selectedCycle = inventory_service.get_last_selected_cycle();
```

Comment:
You might want to add a fallback to the previous `$scope.cycles[0]`. Since there might not be anything in the cache, it'll get the import modal in a weird state in that case.

![Screen Shot 2020-01-30 at 6 09 09 PM](https://user-images.githubusercontent.com/30608004/73504171-b4241480-438b-11ea-9a08-38f91d9e1c4b.png)

There's probably a few examples of a similar pattern in the codebase somewhere.

Reply:
Note, ☝️ happens consistently when you switch orgs. I'm sure it happens when you've just got nothing cached as well. 

There's a localstorage pattern in SEED that caches historical "choices" by, say, org, if you can find it, that might be even better to implement. But I'm not convinced it's necessary since most users likely aren't jumping back and forth between organizations like developers might be.